openslides/users/pdf.py

```diff
@@ -23,8 +23,8 @@ def users_to_pdf(pdf):
     """
     Create a list of all users as PDF.
     """
-    data = [['#', _('Title'), _('Last Name'), _('First Name'),
-             _('Structure level'), _('Group')]]
+    data = [['#', _('Participant number'), _('Title'), _('Last Name'),
```

Comment:
I added some improvements for users list pdf in a new pull request, see https://github.com/matakuka/OpenSlides/pull/1. Please merge it first.